app/src/main/res/layout/item_stream_segment.xml

```diff
@@ -48,6 +48,18 @@
             app:layout_constraintTop_toTopOf="parent"
             tools:text="Lorem ipusum is widely used to create long sample text which is used here too" />
```

Comment:
This now looks like this in the preview:

![image](https://user-images.githubusercontent.com/16943720/114320542-91a49400-9b16-11eb-8a9a-b4b3ffe0510b.png)

To fix the rendering, you could add `tools:maxLines="1"` here.